crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1118,14 +1283,26 @@ where
                 "Received signed Effects but not with a quorum {:?}", non_quorum_effects
             );
 
+            // Safe to unwrap because we know that there is at least one entry in the map
+            // from the check above.
+            let (_most_staked_effects_digest, (_, most_staked_effects_digest_stake)) =
+                non_quorum_effects
+                    .iter()
+                    .max_by_key(|&(_, (_, stake))| stake)
+                    .unwrap();
+            // We check if we have enough retryable stake to get quorum for the most staked
+            // effects digest.
+            state.retryable = most_staked_effects_digest_stake + self.get_retryable_stake(&state)
+                >= self.committee.quorum_threshold();
```

Comment:
cool! When our safety assumption holds, it should always be retriable. So we could at least log an error here or even just panic when it really occurs